hir/src/dataflow.rs

```diff
@@ -127,6 +127,13 @@ impl DataFlowGraph {
         self.values[v].ty()
     }
 
+    pub fn value_span(&self, v: Value) -> SourceSpan {
```

Comment:
There is also `ValueData::span()`, which now seems redundant.

Reply:
Yeah, since it only provides the span for a single variant, it probably can go away